feat(cr): add troubleshooting index (#4589)
scaleway/docs-content: pages/container-registry/troubleshooting/index.mdx

---
meta:
  title: Container Registry - Troubleshooting
  description: Find troubleshooting resources for resolving issues with Container Registry.
content:
  paragraph: Find troubleshooting resources for resolving issues with Container Registry.
dates:
  posted: 2025-03-10
categories:

>   - serverless

Reviewer: ```suggestion
  - containers
```

---

<ProductHeader
  productName="Container Registry troubleshooting"
  productLogo="containerRegistry"

>   description="Troubleshoot common issues with Scaleway's Container Registry. Find solutions to help resolve problems and get started."

Reviewer 2: ```suggestion
  description="Learn how to fix the most common issues with Scaleway's Container Registry."
```
